arsceneview/src/main/java/io/github/sceneview/ar/interaction/ArNodeManipulator.kt

```diff
@@ -58,8 +61,13 @@ open class ArNodeManipulator(
         selectedNode?.takeIf {
             currentGestureTransform == EditableTransform.POSITION && it.positionEditable
         }?.let { node ->
-            node.anchor()
+            lastHitResult?.takeIf { it.isTracking }?.let {
+                node.anchor = it.createAnchor()
+                node.anchor
+            }
+
             currentGestureTransform = null
+            lastHitResult = null
```

Comment:
I think we should check the return values in all `let` blocks since they are compared with `null`, however, `null` is also returned in a positive case here.

Reply:
If we are only interested in `this` value, we can use `also` or `apply`.